packages/salesforcedx-slds-linter/src/client/index.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import { nls } from '../messages';
+
+import {
+  commands,
+  ExtensionContext,
+  window
+} from 'vscode';
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TextEdit,
+  TransportKind
+} from 'vscode-languageclient';
+
+export function createLanguageServer(
+  context: ExtensionContext
+): LanguageClient {
+  // The server is implemented in node
+  const serverModule = context.asAbsolutePath(
+    path.join(
+      'node_modules',
+      '@salesforce',
+      'salesforcedx-slds-linter',
+      'out',
+      'src',
+      'server',
+      'index.js'
+    )
+  );
+  // The debug options for the server
+  const debugOptions = { execArgv: ['--nolazy', '--debug=6009'] };
+
+  // If the extension is launched in debug mode then the debug server options are used
+  // Otherwise the run options are used
+  const serverOptions: ServerOptions = {
+    run: { module: serverModule, transport: TransportKind.ipc },
+    debug: {
+      module: serverModule,
+      transport: TransportKind.ipc,
+      options: debugOptions
+    }
+  };
+
+  // Options to control the language client
+  const clientOptions: LanguageClientOptions = {
+    // Register the server for plain text documents
+    documentSelector: ['plaintext', 'html'],
```

Comment:
@ayeshakmaz - I think we only want HTML, right? No need for it to scan plaintext?

Reply:
Yeah, we can take out the plain text option